src/Uno.UI.RuntimeTests.Engine-dotnet-build.slnf

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We should also be adding another one that validates the UWP/WinUI build and that only builds those two.